mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule OpenApiSpex.Mixfile do
   use Mix.Project
 
-  @version "3.1.0"
```

Comment:
This should stay the same version.

Reply:
👍 this is just a relic of the hex issues I was having, I'll get this and the unnecessary formatting changes removed before I remove the WIP.